### PR TITLE
Nashmeyah branch

### PR DESCRIPTION
**Pull request by Nashmeyah**
Deleted player file

### CODE REVIEW

**Comment by BenWix** [comment]
Here I can write comments about this pull request and we can talk about it

**Comment by Nashmeyah** [comment]
no

**Review by BenWix — COMMENTED** [review]
These changes look good

**Review by BenWix — APPROVED** [review]
Approved

**Review by BenWix — DISMISSED** [review]
Make more comments

File: src/games/blackJack/BlackJack.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from "react";
 
 import PlayerHand from "./components/PlayerHand";
 import PlayerActions from "./components/PlayerActions";
-import PlayerScore from "./components/PlayerScore";
```
Comment (BenWix) [inline src/games/blackJack/BlackJack.js]: can we delete this line?

**Review by BenWix — COMMENTED** [review]
check this out

File: src/games/blackJack/BlackJack.js (hunk 1 above)
Comment (Nashmeyah) [reply src/games/blackJack/BlackJack.js]: Deleted

**Review by BenWix — APPROVED** [review]

**Pull request merged**